api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -289,6 +303,72 @@ private RdrWorkspace toRdrWorkspace(DbWorkspace dbWorkspace) {
     return rdrWorkspace;
   }
 
+  WorkspaceDemographic toWorkspaceDemographics(Set<SpecificPopulationEnum> dbPopulationEnumSet) {
+    WorkspaceDemographic demographic = new WorkspaceDemographic();
+
+    demographic.setAccessToCare(
+        dbPopulationEnumSet.contains(SpecificPopulationEnum.ACCESS_TO_CARE)
+            ? WorkspaceDemographic.AccessToCareEnum.NOT_EASILY_ACCESS_CARE
+            : WorkspaceDemographic.AccessToCareEnum.UNSET);
```

Comment:
What are these setters doing? I', a little confused why it's converting to names that are different?

Reply:
Its because WorkspceDemographic is RDR object so its converting it from what we have in our DB to Workspace Demographic. Our DB store all the population selection as an ArrayList while RDR has different attributes for each cateogory, I can change the name in the yaml to make it easier to understand.